packages/tradershub/src/features/cfd/screens/CFDSuccess/CFDSuccess.tsx

```diff
@@ -8,26 +8,29 @@ import MT5SwapFreeSuccess from '../../../../public/images/cfd/mt5-swap-free-succ
 import CheckMark from '../../../../public/images/checkmark.svg';
 import { TMarketTypes, TPlatforms } from '../../../../types';
 
-type TProps = {
+type TCFDSuccess = {
     description: string;
-    renderButtons?: () => React.ReactNode;
+    renderButtons?: () => ReactNode;
 } & (
     | {
+          displayBalance: string;
+          landingCompany?: string;
           marketType: TMarketTypes.SortedMT5Accounts;
           platform: 'mt5';
+          title: string;
       }
     | {
           marketType?: never;
           platform: Exclude<TPlatforms.All, 'mt5'>;
       }
 );
 
-type PlatformDetails = {
-    all?: { icon: React.ReactNode };
-    financial?: { icon: React.ReactNode };
-    icon?: React.ReactNode;
-    synthetic?: { icon: React.ReactNode };
-};
+type PlatformDetails = Partial<{
```

Comment:
Should follow the one naming convention.
Either all types should have `T` prefix or none would have.